v3/lints/cabf_br/lint_aia_must_contain_permitted_access_method.go

```diff
@@ -0,0 +1,113 @@
+package cabf_br
+
+/*
+ * ZLint Copyright 2024 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"fmt"
+
+	"github.com/zmap/zcrypto/encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+type bRAIAAccessMethodAllowed struct{}
+
+/************************************************************************
+7.1.2.7.7 Subscriber Certificate Authority Information Access
```

Comment:
:pray: thank for putting the full citation in comments. Perhaps I should start asking of this more regularly since I often spend a good chunk of my time hunting these down so that I can double check the interpretation.